apps/website/.vuepress/public/core-usage-demos/input-group/control-actions.html

```diff
@@ -0,0 +1,42 @@
+<form>
```

Comment:
The reason these are in `public` is back to when we would dynamically load these files, they don't have to be here anymore.